PluginsAndFeatures/azure-toolkit-for-intellij/.idea/gradle.xml

```diff
@@ -28,11 +29,13 @@
             <option value="$PROJECT_DIR$/azure-intellij-plugin-hdinsight" />
             <option value="$PROJECT_DIR$/azure-intellij-plugin-hdinsight-base" />
             <option value="$PROJECT_DIR$/azure-intellij-plugin-hdinsight-lib" />
+            <option value="$PROJECT_DIR$/azure-intellij-plugin-integration-services" />
```

Comment:
Ideally, `.idea` directory should be in .gitignore. This file should not be checked in.